CODE_OF_CONDUCT-it.md

```diff
@@ -0,0 +1,28 @@
+# Codice di Condotta dei collaboratori
```

Comment:
Take a look to this:

https://www.contributor-covenant.org/it/version/1/3/0/code-of-conduct/code_of_conduct.md

Reply:
They are pretty similar, do you want that I change my translation with the official one?

Reply:
Original is always the best option, if you are sure that meaning is the same

Reply:
Done, thanks for the help